src/services/mongoose/users.ts

```diff
@@ -57,6 +74,21 @@ export const UsersRepository = (): IUsersRepository => {
     }
   }
 
+  const persistNewKratosUser = async ({
+    userId,
+  }: {
+    userId: string
```

Comment:
maybe we should change param name to something like `kratosUserId` or `oauthUserId` because `userId` is used in other service methods (this apply to all kratos related methods)